jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -552,13 +552,13 @@ fi
 IGN_PHYSICS_DART_FROM_PKGS="true"
```

Comment:
I think this needs to change, see how it's used: https://github.com/gazebo-tooling/release-tools/commit/eb1e2dd092101af977d7f571be6306a075fe46a8

```suggestion
GZ_PHYSICS_DART_FROM_PKGS="true"
```